reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md

```diff
@@ -0,0 +1,133 @@
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: Arnold Beilmann
+
+version: v7.0.0
+
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/solution-guide #TODO
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.svg
+
+related_links:
+  - title: 'SAP in IBM Cloud documentation'
+    url: 'https://cloud.ibm.com/docs/sap'
+    description: 'SAP in IBM Cloud documentation.'
+  - title: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+    url: 'https://url' #TODO
+    description: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+
+use-case: ITServiceManagement
+
+industry: Technology
+
+compliance: SAPCertified
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Power infrastructure for deployable architectures - variation 'PowerVS workspace'
+{: #deploy-arch-ibm-pvs-inf-standard}
+{: toc-content-type="Power infrastructure for deployable architectures - variation 'PowerVS workspace'"}
+{: toc-industry="Technology"}
+{: toc-use-case="ITServiceManagement"}
+{: toc-compliance="SAPCertified"}
+
+Building upon the VPC service that were previously created when you deployed Secure infrastructure on VPC for regulated industries, Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
+
+Additional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally.
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace'.](deploy-arch-ibm-pvs-inf-standard.svg "Architecture diagram"){: caption="Figure 1. Single-zone PowerVS workspace accessable over secure landing zone" caption-side="bottom"}
+
+## Design requirements
+{: #design-requirements}
+
+![Design requirements for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace'.](heatmap.svg "Design requirements"){: caption="Figure 1. Scope of the solution requirements" caption-side="bottom"}
+
+IBM Cloud® Power Virtual Servers (PowerVS) is a public cloud offering that lets an enterprise establish its own private IBM Power computing environment on shared public cloud infrastructure. PowerVS is logically isolated from all other public cloud tenants and infrastrucutre components, creating a private, secure place on the public cloud. The reference architecture for 'Power infrastructure for deployable architectures' - variation 'PowerVS workspace' is designed to provide a framework for building a PowerVS offering according to the best practices and requirements using IBM Cloud® deployable architectures framework.
```

Comment:
```suggestion
IBM Cloud® Power Virtual Servers (PowerVS) is a public cloud offering that an enterprise can use to establish its own private IBM Power computing environment on shared public cloud infrastructure. PowerVS is logically isolated from all other public cloud tenants and infrastructure components, creating a private, secure place on the public cloud. This reference architecture provides a framework to build a PowerVS offering according to the best practices and requirements from the IBM Cloud® deployable architectures framework.
```

fixes typo: infrastrucutre, avoids the anthropomorphism (product lets you) and simplifies the last sentence by shortening it.